### ISSUE TITLE
Power header suggestion

### ISSUE THREAD

**Issue by SanguineBrah** [scrapcomputing/MCEBlaster#4]
If you ever decide to make another revision of the PCBs, I would recommend using something like a Molex KK or floppy-style berg header footprint for powering the board. This should make it difficult to plug the power in backwards by mistake.

**Comment by scrapcomputing** [comment]
Yeah, the floppy-style connector would be ideal. Do you happen to know of any part numbers for it?

**Comment by SanguineBrah** [comment]
The straight header is TE Connectivity AMP Connectors 171825-4 and the right angle version is 171826-4. They are a little strange because the pin spacing is 2.5mm instead of the usual 2.54mm (0.1'').

**Comment by scrapcomputing** [comment]
Thanks!
